package.json

```diff
@@ -20,11 +20,13 @@
    "dependencies": {
       "@actions/core": "^1.11.1",
       "@actions/exec": "^1.0.0",
-      "@actions/tool-cache": "^2.0.2"
+      "@actions/tool-cache": "^2.0.2",
+      "@octokit/rest": "^22.0.0",
+      "ncc": "^0.3.6"
```

Comment:
ncc is already included in the devDependencies as `@vercel/ncc` so we don't need it here